File: app/src/main/java/com/absinthe/anywhere_/adapter/gift/BaseGiftNode.java
package com.absinthe.anywhere_.adapter.gift;

import com.chad.library.adapter.base.entity.node.BaseNode;

import org.jetbrains.annotations.Nullable;

import java.util.List;

public class BaseGiftNode extends BaseNode {

    private String msg;
    public int type;

    @Nullable
    @Override
    public List<BaseNode> getChildNode() {
        return null;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }
}

File: app/src/main/java/com/absinthe/anywhere_/adapter/gift/BaseGiftNode.kt
package com.absinthe.anywhere_.adapter.gift

import com.chad.library.adapter.base.entity.node.BaseNode

open class BaseGiftNode : BaseNode() {
    
    var msg: String? = null
    var type = 0
    
    override val childNode: MutableList<BaseNode>?
        get() = null
}